src/admin/components/card/order.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import { PieChart, Pie, Cell, Tooltip, ResponsiveContainer } from 'recharts';
+
+const Order = () => {
+  const data = [
```

Comment:
Extract this data to a separate file as well

Reply:
done